docs/ManualDeploymentGuideScalarDBServerAKS.md

```diff
@@ -0,0 +1,174 @@
+  In this guide, it is used to deploy Scalar DB server helm charts to the AKS cluster.
+
+
+### Steps
+
+1. Download the following Scalar DB configuration files from [the scalar-kubernetes repository](https://github.com/scalar-labs/scalar-kubernetes/tree/master/conf). Note that they are going to be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardb-custom-values.yaml
+    
+2. Update the database configuration in  scalardb-custom-values.yaml  sections as specified in [configure Scalar DB guide](./ConfigureScalarDB.md).
+
+3. Run the Helm commands on the bastion to install Scalar DB server on AKS.
+
+   ```console
+    # Add Helm charts
+      helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+      helm search repo scalar-labs
+    
+    # Install Scalar DB with a release name `my-release-scalardb`
+      helm upgrade --version <chart version> --install my-release-scalardb scalar-labs/scalardb --namespace default -f scalardb-custom-values.yaml
+   ```
+
+
+## Step 5. Monitor the cluster
+
+It is critical to actively monitor the overall health and performance of a cluster running in production.
+This section shows how to configure container insights for the AKS cluster, Container insights gives you performance visibility by collecting memory and processor metrics from controllers, nodes, and containers.
+Container insights collects container logs also for log monitoring.
+For more information on the container insights you can follow the [official guide](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-overview).
+
+## Recommendations
+
+* You should configure alerting for the AKS cluster on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-metric-alerts#enable-alert-rules)
+
+### Steps
+
+* Enable monitoring of Azure Kubernetes Service on the basis of [the official guide](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-enable-existing-clusters).
+
+## Step 6. Checklist for confirming Scalar DB server deployment
+
+You can check if the pods and the services are properly deployed by running the `kubectl get pods,services -o wide` command on the bastion.
+* You should confirm the status of all scalardb and envoy pods are `Running`.
+
+```console
+kubectl get pods,services -o wide
+NAME                                             READY   STATUS    RESTARTS   AGE   IP             NODE                                   NOMINATED NODE   READINESS GATES
+pod/my-release-scalardb-64f7c9bbc-hqv2q          1/1     Running   0          24s   10.43.41.95    aks-scalardbpool-67464374-vmss000001   <none>           <none>
+pod/my-release-scalardb-64f7c9bbc-ksmz7          1/1     Running   0          24s   10.43.40.247   aks-agentpool-67464374-vmss000002      <none>           <none>
```

Comment:
Please add affinity otherwise it will be deployed in agentpool.

Reply:
I will update the configuration file with affinity. I will update once it is tested and confirmed.

Reply:
Updated document on the suggestion